src/dmd/func.d

```diff
@@ -205,6 +205,17 @@ enum FUNCFLAG : uint
     printf           = 0x200,  /// is a printf-like function
     scanf            = 0x400,  /// is a scanf-like function
     noreturn         = 0x800,  /// the function does not return
+    NRVO             = 0x1000, /// Support for named return value optimization
+    naked            = 0x2000, /// The function is 'naked' (see inline ASM)
+    generated        = 0x4000, /// The function is compiler generated (e.g. `opCmp`)
+    introducing      = 0x8000, /// If this function introduces the overload set
+    semantic3Errors  = 0x10000, /// If errors in semantic3 this function's frame ptr
+    noEH             = 0x20000, /// No exception unwinding is needed
+    inferRetType     = 0x40000, /// Return type is to be inferred
+    dualContext      = 0x80000, /// has a dual-context 'this' parameter
+    hasAlwaysInline  = 0x100000, /// Contains references to functions that must be inlined
+    CRTCtor          = 0x200000, /// Has attribute pragma(crt_constructor) (set in glue)
+    CRTDtor          = 0x400000, /// Has attribpute pragma(crt_destructor) (set in glue)
```

Comment:
Missing from headers.

Reply:
This is not part of the C++ headers though (FUNCFLAG itself)

Reply:
Ok, so just the accessors are enough then to allow gdc/ldc to continue querying these.